sdk/identity/identity/test/manual/README.md

```diff
@@ -2,7 +2,7 @@
 
 This package contains some simple manual verification for the use of Azure
 Identity with other Azure SDK libraries in the browser.  For now, it just tests
-that the `InteractiveBrowserCredential` works with the `@azure/keyvault-keys`
+that the `InteractiveBrowserCredential` works with the `@azure/service-bus`
```

Comment:
Ran into this..

> `Access to fetch at 'https://login.microsoftonline.com/<tenant-id>/oauth2/v2.0/token' from origin 'http://localhost:8080' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.`

Why don't I run into the same error when I bundling and serving with "parcel"?

Reply:
So, running with a new AAD application worked fine.
Might be that the older AAD apps are not supported the same way?

Reply:
They should work too 👍 the app that I have (the one you saw me run this through) is an older one. Can you try again?

Reply:
It failed again, it is not a blocker of this PR. Based on our discussion offline, we don't know what's causing it.
Let's investigate further once the MSAL people get back to you on the issue.